shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -548,14 +543,14 @@ - (void)surfaceUpdated:(BOOL)appeared {
   // thread.
   if (appeared) {
     [self installFirstFrameCallback];
-    [_engine.get() platformViewsController] -> SetFlutterView(_flutterView.get());
-    [_engine.get() platformViewsController] -> SetFlutterViewController(self);
-    [_engine.get() platformView] -> NotifyCreated();
+    [_engine.get() platformViewsController]->SetFlutterView(_flutterView.get());
```

Comment:
Is this stuff because the newest toolchain brought a new `clang-format` along with it? If possible, can you submit this without this bit. Lets format the repo again so PRs are not polluted with these changes.

Reply:
I believe so.  Without this, the format ci job fails.

Reply:
My formatter always clobbered these, nice to have CI in alignment with my tools.

Reply:
I have to pull these out again because the toolchain roll got reverted.  When I land it again, I'll land a full reformat with or after it.